community/gill-jupiter-swap/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "gill-jupiter-swap",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
we are missing a few properties to generate the metadata. please refer to this part

https://github.com/solana-foundation/templates/blob/main/COMMUNITY_TEMPLATE_GUIDE.md

- a better display name
- a usecase. I'd select "swaps" 
- a og-image (see how the script helps generate them in the shared docs)